### ISSUE TITLE
-Wsometimes-uninitialized in drivers/mtd/ubi/wl.c

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#383]
reported in #381 
```
drivers/mtd/ubi/wl.c:1514:6: error: variable 'err' is used uninitialized whenever 'if' condition is false [-Werror,-Wsometimes-uninitialized]
```

**Comment by nickdesaulniers** [comment]
`err` should be initialized to `0`.  Otherwise `if (err == UBI_IO_BITFLIPS || force) {` is potentially a comparison against uninitialized stack memory (UB).

**Comment by nickdesaulniers** [comment]
`ubi_bitflip_check` is only in linux-next.

**Comment by nathanchance** [comment]
Dan Carpenter sent a fix for this independently: https://lore.kernel.org/linux-mtd/20190228053551.GB3253@kadam/

I'll track this one.

**Comment by nathanchance** [comment]
This is fixed in `next-20190306`: https://git.kernel.org/next/linux-next/c/5578e48e5c0bf4684e68ae08caa2293bfb3f5307
